model_bakery/__init__.py

```diff
@@ -1,6 +1,6 @@
 __version__ = '1.6.0'
-__title__ = 'model_mommy'
-__author__ = 'Vanderson Mota'
+__title__ = 'model_bakery'
+__author__ = 'Bernardo Fontes'
```

Comment:
Do we really need this? Also, the original creator is Vanderson, so maybe would be fair to leave this field out.

Reply:
I talked with him about it and he was good. But if it opens space for any kind of confusion, I agree with you!